streaming/index.js

```diff
@@ -148,14 +132,19 @@ const pgConfigFromEnv = (env) => {
  */
 const redisConfigFromEnv = (env) => {
   const redisNamespace = env.REDIS_NAMESPACE || null;
+  let path = null;
+  if (env.REDIS_URL.startsWith('unix://')) {
+    path = env.REDIS_URL.slice(7);
+  }
 
   const redisParams = {
-    socket: {
-      host: env.REDIS_HOST || '127.0.0.1',
-      port: env.REDIS_PORT || 6379,
-    },
-    database: env.REDIS_DB || 0,
-    password: env.REDIS_PASSWORD || undefined,
+    host: env.REDIS_HOST || '127.0.0.1',
+    port: env.REDIS_PORT || 6379,
```

Comment:
ditto, if path, then I don't think host & port should be set?

Reply:
ioredis will use `path` over `host` and `port` itself :)

Reply:
Let's document that in the code, just to avoid confusion later. e.g., have host, port and path lines next to each other, and then describe that behaviour.